test/swingsetTests/scaffolding/contractHost/vat-host.js

```diff
@@ -3,7 +3,7 @@
 import harden from '@agoric/harden';
 import evaluate from '@agoric/evaluate';
 
-import { makeContractHost } from '../../core/contractHost';
+import { makeContractHost } from '../../../../core/contractHost';
 
 function setup(syscall, state, helpers) {
```

Comment:
I wonder whether there's a simple way to have a master copy of vat-host and vat-mint, and have all the tests that need them share it rather than having separate copies in every directory.

Reply:
What do you think about adding this as an issue, but not doing it in this PR?